### ISSUE TITLE
Can not open python file

### ISSUE THREAD

**Issue by SongZihui-sudo** [truecrab/VSCode_Extension_Verilog#11]
![image](https://github.com/truecrab/VSCode_Extension_Verilog/assets/77034643/660e732a-1a27-493e-8345-418640c8e97a)

**Comment by Mudassir10X** [comment]
yes there is an issue that whenever the testbench command is started, the terminal prompt sent has

~/.vscode/extensions/truecrab.verilog-testbench-instance-0.0.5/out\vTbgenerator.py

The \ after out should be changed to /to make it work properly
![image](https://github.com/truecrab/VSCode_Extension_Verilog/assets/140052763/fab3db2e-f9c4-40b9-b8fe-6e1193373004)
